### PR TITLE
 [Full stack build] Enabled building libva

### PR DESCRIPTION
**Pull request by eponomar**

### CODE REVIEW

File: common/component_checker.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        log.info(SKIP_BUILDING_DEPENDENCY_PHRASE)
+    else:
+        log.info(f'Directory {component_dir} is not exist')
+
+
+def main():
+    parser = argparse.ArgumentParser(prog="component_checker.py",
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('-p', '--path-to-manifest', help='Path to manifest file', required=True)
+    parser.add_argument('-c', '--component-name', help='Name component to checking', required=True)
```
Comment (azhogov) [inline common/component_checker.py]: Component name to check

File: common/component_checker.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        repository.revision,
+        component.product_type,
+        Build_type.RELEASE.value,
+        product=component_name)
+    if component_dir.exists():
+        log.info(f'Directory {component_dir} is exist')
+        # This is stop phrase for buildbot to skip all build stages
+        log.info(SKIP_BUILDING_DEPENDENCY_PHRASE)
+    else:
+        log.info(f'Directory {component_dir} is not exist')
```
Comment (azhogov) [inline common/component_checker.py]: doesn't

File: common/component_checker.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    repository = component.get_repository(component_name)
+    component_dir = MediaSdkDirectories.get_build_dir(
+        repository.branch,
+        Build_event.COMMIT.value,
+        repository.revision,
+        component.product_type,
+        Build_type.RELEASE.value,
+        product=component_name)
+    if component_dir.exists():
+        log.info(f'Directory {component_dir} is exist')
```
Comment (azhogov) [inline common/component_checker.py]: doesn't 
or was not found

File: common/component_checker.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        # This is stop phrase for buildbot to skip all build stages
+        log.info(SKIP_BUILDING_DEPENDENCY_PHRASE)
+    else:
+        log.info(f'Directory {component_dir} is not exist')
+
+
+def main():
+    parser = argparse.ArgumentParser(prog="component_checker.py",
```
Comment (azhogov) [inline common/component_checker.py]: it is good to add some help string

File: common/component_checker.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('-p', '--path-to-manifest', help='Path to manifest file', required=True)
+    parser.add_argument('-c', '--component-name', help='Name component to checking', required=True)
+    args = parser.parse_args()
+
+    configure_logger()
+    log = logging.getLogger('component_checker.main')
+
+    # Exit code of this script must be 0, because its status can affect Buildbot pipeline.
```
Comment (azhogov) [inline common/component_checker.py]: probably it should be in Buildbot utils.

File: common/component_checker.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2019 Intel Corporation
```
Comment (azhogov) [inline common/component_checker.py]: I see no pylint run for this script here: http://mediasdk.intel.com/buildbot/#/builders/17/builds/1097

File: common/component_checker.py [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+#
```
Comment (azhogov) [inline common/component_checker.py]: script name is not good

File: common/component_checker.py (hunk 6 above)
Comment (eponomar) [reply common/component_checker.py]: File was not changed in last commit, see pre-commit check for first commit http://mediasdk.intel.com/buildbot/#/builders/17/builds/1096

File: common/component_checker.py [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+        repository.branch,
+        Build_event.COMMIT.value,
+        repository.revision,
+        component.product_type,
+        Build_type.RELEASE.value,
+        product=component_name)
+    if component_dir.exists():
+        log.info(f"Directory {component_dir} exist")
+        # This is stop phrase for buildbot to skip all build stages
+        log.info(SKIP_BUILDING_DEPENDENCY_PHRASE)
```
Comment (azhogov) [inline common/component_checker.py]: should be json data like <BUILDS_STATUS>{json data}<BUILDS_STATUS>
ok, let's improve later

File: common/component_checker.py [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+    repository = component.get_repository(component_name)
+    component_dir = MediaSdkDirectories.get_build_dir(
+        repository.branch,
+        Build_event.COMMIT.value,
+        repository.revision,
+        component.product_type,
+        Build_type.RELEASE.value,
+        product=component_name)
+    if component_dir.exists():
+        log.info(f"Directory {component_dir} exist")
```
Comment (azhogov) [inline common/component_checker.py]: exists

**Pull request merged**